README.md

```diff
@@ -195,29 +195,31 @@ zlib
 
 ## Testing
 
-### `Golden` Tests
-Before you run `golden` tests you need to obtain a `coffer` executable. You can get it in two ways:
-1. Cabal
-```shell
-$ cabal install
-```
-2. Stack
-```shell
-$ stack install
-```
-These commands would build the project and add a symblink for a `coffer` executable.
+The package contains 4 test suites:
+1. A unit tests suite for the coffer library
+1. An integration test suite for the Web API.
```

Comment:
Mb we can make it like:
2. `server-integration` -- an integration test suite for the Web API.